packages/styles/src/sass/components/footer/_internal.scss

```diff
@@ -0,0 +1,193 @@
+@use '../../tokens/breakpoints' as *;
+
+// Styling for Link pattern (Dark)
+@mixin footer-links {
+  color: var(--cbp-color-interactive-primary-light);
+
+  & > i {
+    margin-right: var(--cbp-space-1x);
+  }
+
+  &:hover {
+    color: var(--cbp-color-interactive-primary-base);
+  }
+
+  &:focus {
+    color: var(--cbp-color-interactive-focus-light) !important;
```

Comment:
What is overriding this that you need `!important`? Any place you have `!important`, see if you can get rid of it either with ordering of the rules (cascade) or specificity.

Reply:
re-ordered pseudo classes in the mixin, will be taken out in the next buttons PR and imported over.